midi_app_controller/controller/connected_controller.py

```diff
@@ -0,0 +1,363 @@
+import rtmidi
+import time
+
+from typing import List, Dict
+from pydantic import BaseModel
+
+from midi_app_controller.models.controller import Controller
+from ..actions.actions_handler import ActionsHandler
+from .controller_constants import ControllerConstants
+
+# Leave uncommented for debug purpouses
+# import sys
+
+
+def midi_callback(message, cls):
```

Comment:
Why do you pass `cls` instead of simply making this function a method of `ConnectedController`?

Reply:
I tried this approach in many different ways, but couldn't make it work for some reason.